### PR TITLE
Add interview with Mahana

### PR DESCRIPTION
**Pull request by Crunch09**
Commiting on behalf of Mahana Mansfield / James Dance

### CODE REVIEW

File: Gemfile.lock [hunk 1]
```diff
@@ -265,27 +272,27 @@ GEM
...
     unicode-display_width (1.7.0)
-    zeitwerk (2.4.0)
+    zeitwerk (2.4.2)
 
 PLATFORMS
   ruby
 
 DEPENDENCIES
-  github-pages (= 207)
+  github-pages (= 212)
```
Comment (cedricbastin) [inline Gemfile.lock:295]: Thanks so much for updating these, I hope this will fix my local Docker setup!! 🍀

**Review by cedricbastin — APPROVED** [review]
Thanks a lot

File: _posts/2021-03-19-a-glimpse-into-data-science.md [hunk 2]
```diff
@@ -0,0 +1,41 @@
...
+layout: post
+title:  "A glimpse into Data Science at Deliveroo"
+authors:
+  - "Mahana Mansfield"
+excerpt: >
+  Learn more about Data Science at Deliveroo from the eyes of Mahana Mansfield, DS Director.
+---
+
+
+> Mahana, let’s start with a bit about you. You initially trained to do Judo at the London Olympics as a professional athlete, then got a PhD in Maths, then worked as a Software Engineer at Ocado, then became a Head of Data Science at The Guardian, and now you’re our Data Science Director. What a background! What brought you to Deliveroo?
```
Comment (cedricbastin) [inline _posts/2021-03-19-a-glimpse-into-data-science.md:11]: not sure about the quoted questions, maybe _italic_ or **bold** would look better

Comment (cedricbastin) [reply _posts/2021-03-19-a-glimpse-into-data-science.md:11]: here is how it looked in LinkedIn: https://www.linkedin.com/pulse/glimpse-data-science-deliveroo-through-eyes-mahana-ds-dance-he-him-/?trackingId=eAkZkvBxR1C7zjVXnFdS3g%3D%3D

Comment (Crunch09) [reply _posts/2021-03-19-a-glimpse-into-data-science.md]: Thanks! I made it italic *and* bold now and I agree that it looks better

File: Gemfile.lock (hunk 1 above)
Comment (cedricbastin) [reply Gemfile.lock:295]: It runs!! `http://localhost:4000`

**Pull request merged**